src/GovUk.Education.ExploreEducationStatistics.Data.Api/Models/PermalinkEntity.cs

```diff
@@ -0,0 +1,21 @@
+namespace GovUk.Education.ExploreEducationStatistics.Data.Api.Models
+{
+    using System;
+    using Microsoft.Azure.Cosmos.Table;
+    
+    public class PermalinkEntity : TableEntity
+    {
+        public PermalinkEntity()
+        {
+            // TODO: im not sure about the partition key, requires some more thought
```

Comment:
Create a Jira ticket for this TODO?

Reply:
Yeah, its on my list of jira tickets to create :)